components/x-teaser/src/Title.jsx

```diff
@@ -28,9 +28,8 @@ export default ({ title, altTitle, headlineTesting, relativeUrl, url, indicators
 			{indicators && indicators.accessLevel === 'premium' ? (
 				<span>
 					{' '}
-					<span className={premiumClass} aria-label="Premium content">
-						Premium
-					</span>
+					<span className={premiumClass}>Premium</span>
```

Comment:
**issue**: this element should be hidden from screenreaders, otherwise they'll read "Premium premium content"